src/armsgs.py

```diff
@@ -20,8 +20,9 @@ def __init__(self, log, debug, last_updated, version, verbose, colors=True):
         ----------
         log : str
           Name of saved log file (no log will be saved if log=="")
-        debug : bool
-          Used for debugging. Should be set to False in all other cases
+        debug : list
```

Comment:
Note, a change needs to be made to the debugmessage() function of this class to accommodate the change of self._debug to a list.